packages/compass-connections/src/connections-manager.ts

```diff
@@ -14,6 +14,7 @@ import { cloneDeep, merge } from 'lodash';
 import { adjustConnectionOptionsBeforeConnect } from '@mongodb-js/connection-form';
 import mongodbBuildInfo from 'mongodb-build-info';
 import { openToast } from '@mongodb-js/compass-components';
+import { createCancelError, isCancelError } from '@mongodb-js/compass-utils';
```

Comment:
Drive-by, we have this helper we use elsewhere to create cancellation errors, using it here removes the need to checking error message to figure out if connection was canceled (which is not the most stable way of doing this)